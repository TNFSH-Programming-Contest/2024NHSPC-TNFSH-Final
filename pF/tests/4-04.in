734114249902 953650
