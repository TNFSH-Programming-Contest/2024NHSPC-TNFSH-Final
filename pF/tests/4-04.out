476824507912523822